code/index.html

```diff
@@ -1,22 +1,163 @@
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The New Forker</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="icon" type="image/png" href="./noun_fork.png"/>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <a href="#"><img class="header-image" src="./The_New_Forker_logo.png" alt="The New Forker logo" /></a>
+    <nav>
+      <a href="#">Books & Culture</a>
+      <a href="#">Crossword</a>
+      <a href="#"><img class="nav-image" src="./The_New_Forker_logo.png" alt="The New Forker logo" /></a>
+      <a href="#">Goings On</a>
+      <a href="#">Fiction & Poetry</a>
+    </nav>
+  </header>
+  <main>
+    <!-- The big news content -->
     <section class="big-news">
-      <!-- The big news content -->
+      <div class="big-news-card">
+        <img src="./COVID-19-adam-niescioruk-Z9arfr0f248-unsplash.jpg" 
+        alt="Black facial mask graffiti on a white wall.">
+        <article class="big-news-article">
+          <a href="#">News</a>
+          <h1>"Det är fredag mina bekanta!"</h1>
+          <p>
+            Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+            Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+            Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+            Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </p>
+        </article>
+      </div>
     </section>
-
-    <section class="">
-      <!-- The other news  -->
+    <!-- The other news  -->
+    <section class="small-news">
+      <div class="small-news-card">
+        <img src="./troopers-jon-tyson-eIhH7RTlTZA-unsplash.jpg" 
+        alt="Image of storm troopers and Anonymous.">
+        <article>
+          <a href="#">Books & Culture</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Ticky DeFan.</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./tiktok.jpeg" 
+        alt="Image of tiktok text written by a typwriter.">
+        <article>
+          <a href="">Crossword</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Fanny Svensson</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./grafitti.jpg" 
+        alt="Colorful grafitti.">
+        <article>
+          <a href="#">Goings On</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
```

Comment:
If there is a line break between content and closing tag, it should be there also for opening tag